packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -64,6 +64,79 @@ abstract class InteractiveInkFeature extends InkFeature {
     _color = value;
     controller.markNeedsPaint();
   }
+
+  /// Draws an ink splash or ink ripple on the passed in [Canvas].
+  ///
+  /// The [transform] argument is the [Matrix4] transform that typically
+  /// shifts the coordinate space of the canvas to the space in which
+  /// the ink circle is to be painted.
+  ///
+  /// [center] is the [Offset] from origin of the canvas where center
+  /// of the circle is drawn
+  ///
+  /// Using [paint] you can specify properties like color, strokewidth, colorFiler, etc
+  /// for painting the circle
+  ///
+  /// [radius] is the radius of circle to be drawn on canvas
+  ///
+  /// [clipCallback] is required by ink effects to obtain the rectangle for the effect,
```

Comment:
I think using the word required comes off too strong, since it implies that the parameter is required.
```suggestion
  /// [clipCallback] is the callback used to obtain the [Rect] used for clipping the ink effect.
```